mdpow/analysis/ensemble.py

```diff
@@ -412,9 +408,9 @@ class in MDAnalysis <https://docs.mdanalysis.org/stable/documentation_pages/anal
 
     Dihedral Analysis Demonstration::
 
-        class DihedralAnalysis(mdpow._ensemble.EnsembleAnalysis):
+        class DihedralAnalysis(mdpow.ensemble.EnsembleAnalysis):
             def __init__(self, DihedralEnsembleGroup):
-                super(DihedralAnalysis, self).__init__(DihedralEnsembleGroup._ensemble())
+                super(DihedralAnalysis, self).__init__(DihedralEnsembleGroup.ensemble())
```

Comment:
Use the ensemble "attribute"
```suggestion
                super(DihedralAnalysis, self).__init__(DihedralEnsembleGroup.ensemble)
```